### PR TITLE
LPS-187013 check all the url subjects inserted on the table

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
https://issues.liferay.com/browse/LPS-187013

Steps to reproduce:

1. Set up a vanilla Liferay DXP 7.1 FP28 environment
2. Add a Message Board on the Home page
3. Add 3 new separate threads: Test, Test and Test 1
4. (bonus) Add 4 new separate threads: hello, hello, hello and hello 2
5. Upgrade to Master

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by Jeremy-Ch** [comment]
The failures are unrelated.

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

File: modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/upgrade/v3_1_0/UrlSubjectUpgradeProcess.java [hunk 1]
```diff
@@ -80,26 +85,27 @@ private void _populateUrlSubject() throws Exception {
...
-					previousURLSubject = curURLSubject;
-					suffix = StringPool.BLANK;
+					Integer count = urlSubjects.get(curURLSubject);
+
+					urlSubjects.put(curURLSubject, count + 1);
+
+					suffix = StringPool.DASH + count;
+
+					urlSubjects.put(curURLSubject + suffix, 1);
 				}
```
Comment (adolfopa) [inline modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/upgrade/v3_1_0/UrlSubjectUpgradeProcess.java:109]: You can simplify this a little bit by using `getOrDefault`:
```
int count = urlSubjects.getOrDefault(curURLSubject, 0);

urlSubjects.put(..., count + 1);

if (count > 0)  {
    suffix = ...;
    ...
}
```

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
rebase and resend with suggested changes.

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:upgrade

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**